src/stats.c

```diff
@@ -7,7 +7,7 @@ void r_stats_init(struct RaucStats *stats)
 	memset(stats, 0, sizeof(*stats));
 
 	stats->min = G_MAXDOUBLE;
-	stats->min = G_MINDOUBLE;
+	stats->max = G_MINDOUBLE;
```

Comment:
Is that correct? at the very least a comment explaining why min = MAX and max = MIN might be in order.

Reply:
Just saw @jluebbe comment above. Still some comment in the code would be nice.

Reply:
These are initial values for incremental comparison. If they were the reversed, they could never change.

Reply:
There is hardly any documentation of these functions at all. But iirc I did some for me during the review.
If I find it, I'll submit :smirk: